build.gradle

```diff
@@ -65,7 +65,7 @@ dependencies {
     implementation "com.squareup.retrofit2:retrofit:2.9.0"
     implementation "com.squareup.retrofit2:converter-gson:2.9.0"
 
-    implementation 'org.json:json:20230227'
+    implementation 'org.json:json:20231013'
```

Comment:
do we need this as part of the licence change?
do we need this at all, what's changed in this library?

Reply:
ah, I see - the vulnerability. It seems important enough to be in its own PR